website/docs/setup/repos.md

```diff
@@ -159,3 +159,95 @@ need to use the following flag with the initial Corso `repo init` command:
 Corso also supports the use of object storage systems with no TLS certificate or with self-signed
 TLS certificates with the `--disable-tls` or `--disable-tls-verification` flags.
 [These flags](../../cli/corso-repo-init-s3) should only be used for testing.
+
+## Filesystem Storage
+
+Corso supports creating a repository on local or network attached filesystem.
+
+:::note
+Filesystem repositories aren't S3 compatible.
```

Comment:
For my own edification, what does this mean?

Reply:
It means we cannot seamlessly transfer repository data between different environments, for e.g. from an on-prem server to a cloud provider, without doing some complex data transformation.

Minio on the other hand is [S3 compatible.](https://min.io/product/s3-compatibility)

Reply:
You should expand on what this means in the note.

Adding something along the lines of "Copying a local repository directly to AWS S3 or the other way around is not supported and will not work."

Not that we are doing it, but we probably should not be encouraging direct repo operations for Minio or other S3 compatible storage either. It should work but is asking for trouble.